scale/scale.go

```diff
@@ -0,0 +1,88 @@
+package scale
+
+import (
+	"github.com/pkg/errors"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/restclient"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+// for testing purposes
+type KubeClient interface {
+	Deployments(namespace string) kclient.DeploymentInterface
+}
+
+type PodAutoScaler struct {
+	Client     KubeClient
+	Max        int
+	Min        int
+	Deployment string
+}
+
+func NewPodAutoScaler(kubernetesDeploymentName string, max int, min int) *PodAutoScaler {
+	config, err := restclient.InClusterConfig()
+	if err != nil {
+		panic("Failed to configure incluster config")
+	}
+
+	k8sClient, err := kclient.New(config)
+	if err != nil {
+		panic("Failed to configure client")
+	}
+
+	return &PodAutoScaler{
+		Client:     k8sClient,
+		Min:        min,
+		Max:        max,
+		Deployment: kubernetesDeploymentName,
+	}
+}
+
+func (p *PodAutoScaler) ScaleUp() error {
+	// TODO: use the same namespace that kube-sqs-autoscaler is using
```

Comment:
This isn't hard to do, just another flag and a env var in the Pod spec.  Might as well throw it in there and save the work later, what do you think?